recipe/meta.yaml

```diff
@@ -20,41 +20,25 @@ requirements:
     - make
     - {{ compiler('cxx') }}
     - {{ compiler('c') }}
-    - qt-main                            # [build_platform != target_platform]
-    - sysroot_linux-64 2.17              # [linux]
-    - {{ cdt('mesa-libgl-devel') }}      # [linux]
-    - {{ cdt('mesa-libegl') }}           # [linux]
-    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ stdlib('c') }}
     - {{ cdt('mesa-dri-drivers') }}      # [linux]
-    - {{ cdt('libdrm-devel') }}          # [linux]
-    - {{ cdt('libglvnd-glx') }}          # [linux]
-    - {{ cdt('libglvnd-egl') }}          # [linux]
+    - qt-main  {{ version }}             # [build_platform != target_platform]
+    - wayland                            # [build_platform != target_platform]
+    - libgl-devel                        # [build_platform != target_platform]
+    - libegl-devel                       # [build_platform != target_platform]
+    - libdrm                             # [build_platform != target_platform]
   host:
     - qt-main  {{ version }}
     - wayland
+    - libgl-devel
+    - libegl-devel
+    - libdrm
     - fontconfig
     - freetype
     - zlib
     - glib
-    # The following X11 dependencies are only needed to build
```

Comment:
see this whole note.

Reply:
even with them included I got
```
XComposite EGL ......................... no
XComposite GLX ......................... no
```

so I removed them, but I don't mind putting them back in

Reply:
Sometimes I dropped into debug mode and used pkg-config --exists --debug to find the missing stuff

Reply:
see https://github.com/conda-forge/qt-wayland-feedstock/pull/25#issuecomment-2440152357 I got XComposite GLX but not EGL to work.